### DISCUSSION TITLE
Customizing the search bar

### DISCUSSION THREAD

**Discussion by datapolitical** [algolia/instantsearch#4616] [Q&A] [answered]
There seems to be little documentation on how to do simple things like changing the size and color of the search box. 

I use a dark theme on my website and I'd like to make a dark background (and light border) for the search box, but the simple things I've tried haven't worked. How do I do that with InstantSearch?

**Comment by eunjae-lee** [comment]
Hi @datapolitical , can you share with us what you've tried but didn't work?

**Comment by datapolitical** [comment, marked as answer]
Sorry I should’ve closed the issue I ended up going in a different direction
